crypto/CMakeLists.txt

```diff
@@ -525,6 +529,13 @@ if(BUILD_TESTING)
     fipsmodule/rand/urandom_test.cc
   )
 
+  # When using CPU Jitter as the entropy source (only in FIPS build)
```

Comment:
I understand that urandom would still be used for the personalization string and additional data, can we still run urandom_test while removing the parts that are executed for getting a seed when defined(JITTER_ENTROPY)?